src/payloads/send_animation.rs

```diff
@@ -0,0 +1,52 @@
+// This file is auto generated by `cg` <https://github.com/teloxide/cg> (e634f65).
+// **DO NOT EDIT THIS FILE**,
+// edit `cg` instead.
+use serde::Serialize;
+
+use crate::types::{ChatId, InputFile, Message, ParseMode, ReplyMarkup};
+
+impl_payload! {
+    /// Use this method to send animation files (GIF or H.264/MPEG-4 AVC video without sound). On success, the sent [`Message`] is returned. Bots can currently send animation files of up to 50 MB in size, this limit may be changed in the future.
+    ///
+    /// [`Message`]: crate::types::Message
+    #[derive(Debug, PartialEq, Eq, Hash, Clone, Serialize)]
+    pub SendAnimation (SendAnimationSetters) => Message {
+        required {
+            /// Unique identifier for the target chat or username of the target channel (in the format `@channelusername`)
+            pub chat_id: ChatId [into],
+            /// Animation to send. Pass a file_id as String to send a video that exists on the Telegram servers (recommended), pass an HTTP URL as a String for Telegram to get a video from the Internet, or upload a new video using multipart/form-data. [More info on Sending Files »]
+            ///
+            /// [More info on Sending Files »]: crate::types::InputFile
+            pub animation: InputFile,
+            /// Animation caption (may also be used when resending videos by _file\_id_), 0-1024 characters after entities parsing
+            pub caption: String [into],
+        }
+        optional {
+            /// Duration of the animation in seconds
+            pub duration: u32,
+            /// Animation width
+            pub width: u32,
+            /// Animation height
+            pub height: u32,
```

Comment:
`u32` -> `i32`

Reply:
This doesn't make any sense - how do you imagine negative sizes of an animation?

Reply:
See https://github.com/teloxide/teloxide-core/pull/25#discussion_r524169388.

Reply:
See https://github.com/teloxide/teloxide-core/pull/25#discussion_r524207000